types.asciidoc

```diff
@@ -2,6 +2,33 @@
 [[types_types]]
 == Types
 
+[[types_class_type]]
+=== class
+
+When a data type is defined as a "class" in the OCPI specification, we mean a type whose possible values are sets of zero or more pairs of a string and another value. The string is known as a "key", "field name", or "property", and the value associated with the key is known as a field value. For each class type, the specification lists which strings are required and allowed to occur as field names in values of that type, and what the types of the field values of these fields should be.
+
+In the serialized JSON form of OCPI messages, class values are serialized as JSON objects.
+
+[[types_enum_type]]
+=== enum
+
+When a data type is defined as an "enum" in the OCPI specification, we mean a type whose possible values are a finite number of strings.
+
+This type is used for class fields where it is clear that there is only a finite set of possible values that is completely known at the time of writing of the specification. An example of a place where this is used is a class field whose possible values are the days of the week.
+
+In the serialized JSON form of OCPI messages, enum values are serialized as JSON strings.
+
+[[types_openenum_type]]
+=== OpenEnum _type_
```

Comment:
Having the data types specified as OpenEnum makes it more difficult to ensure interoperability between parties. Although I understand the need for more flexibility, there is also the risk of getting different values with (more or less) the same meaning. And the risk of losing track of the meaning of a value in the first place. 

An alternative solution might be a DynamicEnum that is still a predefined set of possible values, but with a central list of accepted values. This list could be here on GitHub and if parties feel the need for extra values, they can create a pull request to add a new value. This way there is no need for a new release yet still a centralised list of accepted values.